source/content/integrated-composer.md

```diff
@@ -214,7 +214,8 @@ Merge the changes manually:
 1. Merge in the upstream changes:
 
    ```bash{promptUser: user}
-   git remote add [upstream-name] git@github.com:[org]/[project].git
+   git remote add upstream <upstream_url> && git fetch upstream
+   git merge upstream/master
```

Comment:
@namespacebrian Confirming that this accurately reflects the changes you suggested before moving to copy review- let me know.

Reply:
Actually, replace `<upstream_url>` with `https://github.com/pantheon-upstreams/drupal-recommended.git` in this case.

Reply:
Looks good